Assets/Scripts/Framework/GeoLocation/RouteHighlighter.cs

```diff
@@ -0,0 +1,38 @@
+using System;
+using System.Collections.Generic;
+using System.Collections;
+using UnityEngine;
+
+namespace Framework.GeoLocation
+{
+    public sealed class RouteHighlighter : MonoBehaviour
+    {   
+        [SerializeField] private List<Transform> routePoints;
+        [SerializeField] private LineRenderer lineRenderer;
+
+        private int _routeIndex;
+
+        private void Start()
+        {
+            Invoke("InitLines", 0.1f);
+        }
+
+        private void InitLines()
```

Comment:
```suggestion
        private void InitLine()
```
Het is toch maar 1 line?